apps/emqx_bridge_tdengine/test/emqx_bridge_tdengine_SUITE.erl

```diff
@@ -544,7 +565,156 @@ t_nasty_sql_string(Config) ->
         connect_and_get_payload(Config)
     ).
 
+t_simple_insert(Config) ->
+    connect_and_clear_table(Config),
+    ?assertMatch(
+        {ok, _},
+        create_bridge(Config)
+    ),
+
+    SentData = #{payload => ?PAYLOAD, timestamp => 1668602148000},
+    Request = {send_message, SentData},
+    {_, {ok, #{result := _Result}}} =
+        ?wait_async_action(
+            query_resource(Config, Request),
+            #{?snk_kind := buffer_worker_flush_ack},
+            2_000
+        ),
+    ?assertMatch(
+        ?PAYLOAD,
+        connect_and_get_payload(Config)
+    ).
+
+t_batch_insert(Config) ->
+    connect_and_clear_table(Config),
+    ?assertMatch(
+        {ok, _},
+        create_bridge(Config)
+    ),
+
+    Size = 5,
+    Ts = erlang:system_time(millisecond),
+    {_, {ok, #{result := Result}}} =
+        ?wait_async_action(
+            lists:foreach(
+                fun(Idx) ->
+                    SentData = #{payload => ?PAYLOAD, timestamp => Ts + Idx},
+                    Request = {send_message, SentData},
+                    query_resource(Config, Request)
+                end,
+                lists:seq(1, Size)
+            ),
+
+            #{?snk_kind := buffer_worker_flush_ack},
+            2_000
+        ),
+
+    timer:sleep(200),
+
+    ?assertMatch(
+        [[Size]],
+        connect_and_query(Config, "SELECT COUNT(1) FROM t_mqtt_msg")
+    ).
```

Comment:
To try to be quick:

```suggestion
    ?retry(
      _Sleep = 50,
      _Attempts = 30,
      ?assertMatch(
          [[Size]],
          connect_and_query(Config, "SELECT COUNT(1) FROM t_mqtt_msg")
      )
    ).
```

(similar for the next test case with a sleep)

Reply:
It is cool, thanks you.